cd/scripts/deploy.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+set -e -o pipefail
+export APP_URL="https://github.com/${ORG}/${APP_NAME}/archive/refs/heads/${APP_BRANCH}.tar.gz"
+export USER=$(lsb_release -si | tr [:upper:] [:lower:])
+
+cd /home/$USER && mkdir -p ${APP_NAME} && curl -kLs $APP_URL \
```

Comment:
if I understand well
- we download the entire code of Basegun repo
- for branch ${APP_BRANCH}

why not:
- only download the files we need (Makefile, maybe docker-compose)
- use the branch with tag ${VERSION} ?(if version==latest we try to find which tag it is ?) it seems logical that the Makefile & docker-compose should be issued from same version as the docker images we are going to pull

Reply:
Yes we download the entire code of Basegun for not hardcoding download docker-compose and makefile. Then, github prepare the repo archive natively.

For version, yes, it can be better.